package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "serverless-create-global-dynamodb-table",
-  "version": "3.0.0",
+  "version": "3.1.0",
```

Comment:
can you please delete package-lock.json and rerun npm install so that the new file gets generated and then check that in.

Reply:
@rrahul963 Here I was confused a bit, so I decided to remove "package-lock.json" as you suggested and recreated by using "npm install", and then pushed this change.

Reply:
Oh, I guess I got it now :-)